doc/release/2.11.0.rst

```diff
@@ -0,0 +1,586 @@
+In the example below, the ``select`` operation gets maximum 3 tuples after the specified tuple:
+
+.. code-block:: tarantoolsession
+
+    tarantool> bands.index.primary:select({}, {after = {4, 'The Beatles', 1960}, limit = 3})
+    ---
+    - - [5, 'Pink Floyd', 1965]
+      - [6, 'The Rolling Stones', 1962]
+      - [7, 'The Doors', 1965]
+    ...
+
+The ``after`` option also accepts the position of the tuple represented by the base64 string.
+For example, you can set the ``fetch_pos`` boolean option to ``true`` to return the position of the last selected tuple as the second value:
+
+.. code-block:: tarantoolsession
+
+    tarantool> result, position = bands.index.primary:select({}, {limit = 3, fetch_pos = true})
+    ---
+    ...
+
+Then, pass this position as the ``after`` parameter:
+
+.. code-block:: tarantoolsession
+
+    tarantool> bands.index.primary:select({}, {limit = 3, after = position})
+    ---
+    - - [4, 'The Beatles', 1960]
+      - [5, 'Pink Floyd', 1965]
+      - [6, 'The Rolling Stones', 1962]
+    ...
+
+The new ``after`` and ``fetch_pos`` options are also implemented by the built-in ``net.box`` connector.
+For example, you can use these options to get data :ref:`asynchronously <net_box-is_async>`.
+
+
+
+.. _downgrading_database:
+
+Downgrading a database
+~~~~~~~~~~~~~~~~~~~~~~
+
+The 2.11 version provides the ability to downgrade a database to the specified Tarantool version using the :ref:`box.schema.downgrade() <box_schema-downgrade>` method.
+This might be useful in the case of a failed upgrade.
+
+To prepare a database for using it on an older Tarantool instance, call ``box.schema.downgrade`` and pass the desired Tarantool version:
+
+.. code-block:: tarantoolsession
+
+    tarantool> box.schema.downgrade('2.8.4')
+
+To see Tarantool versions available for downgrade, call ``box.schema.downgrade_versions()``.
+The oldest release available for downgrade is :doc:`2.8.2 </release/2.8.2>`.
+
+
+
+.. _new_bootstrap_strategy:
+
+New bootstrap strategy
+~~~~~~~~~~~~~~~~~~~~~~
+
+In previous Tarantool versions, the :ref:`replication_connect_quorum <cfg_replication-replication_connect_quorum>` option was used to specify the number of nodes to be up and running to start a replica set.
+This option was designed to simplify a replica set bootstrap.
+But in fact, this behaviour brought some issues during a cluster lifetime and maintenance operations, for example:
+
+*   Users who didn't change this option encountered problems with the partial cluster bootstrap.
+*   Users who changed the option encountered problems during the instance restart.
+
+With v2.11, ``replication_connect_quorum`` is deprecated in favor of :ref:`bootstrap_strategy <cfg_replication-bootstrap_strategy>`.
+This option works during a replica set bootstrap and implies sane default values for other parameters based on the replica set configuration.
```

Comment:
"sane" is generally not welcome by big tech [style guides](https://developers.google.com/style/word-list#letter-s).